src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -132,7 +235,18 @@ void PacketDataReporter::OnSrvRecord(SerializedQNameIterator name, const SrvReco
     }
 }
 
-void PacketDataReporter::OnIPAddress(const chip::Inet::IPAddress & addr)
+void PacketDataReporter::OnCommissionableNodeSrvRecord(SerializedQNameIterator name, const SrvRecord & srv)
+{
+    // Host name is the first part of the qname
+    mdns::Minimal::SerializedQNameIterator it = srv.GetName();
+    if (!it.Next())
+    {
+        return;
+    }
+    strncpy(mCommissionableNodeData.hostName, it.Value(), mCommissionableNodeData.kHostNameSize);
```

Comment:
Please use `CommissionableNodeData::kHostNameSize` to make it clear it is a constant not depending on the actual data member.

Reply:
done.